app/reducers/game.js

```diff
@@ -113,6 +112,12 @@ const reducer = (state = initialState, action) => {
         messages: [],
       }
 
```

Comment:
you also need to change backgroundimage to 'night' or 'day' whenever there is a day switch